.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+./public/app.js
```

Comment:
@MarissaShieh Please add a newline break at the end of the file to meet style guide.